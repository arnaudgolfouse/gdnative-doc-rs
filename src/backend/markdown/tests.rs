//! Snapshot tests

use super::*;
use pulldown_cmark::CowStr;

#[test]
fn simple_text() {
    let mut callbacks = MarkdownCallbacks::default();
    let mut res = String::new();
    callbacks.encode(
        &mut res,
        vec![Event::Text(CowStr::Borrowed("hello world !"))],
    );
    insta::assert_display_snapshot!(res)
}

#[test]
fn code_text() {
    let mut callbacks = MarkdownCallbacks::default();
    let mut res = String::new();
    callbacks.encode(
        &mut res,
        vec![Event::Code(CowStr::Borrowed("hello code !"))],
    );
    insta::assert_display_snapshot!(res)
}
