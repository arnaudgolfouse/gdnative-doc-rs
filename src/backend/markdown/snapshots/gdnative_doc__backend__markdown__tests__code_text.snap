---
source: src/backend/markdown/tests.rs
expression: res
---
`hello code !`
